### ISSUE TITLE
Lookup modal has no label

### ISSUE THREAD

**Issue by jcoyne** [LD4P/sinopia_editor#2492]
This is an accessibility concern. All inputs should have visible labels.

**Comment by justinlittman** [comment]
There is no longer a lookup modal.
